### ISSUE TITLE
Image needs 'General' property section and most others

### ISSUE THREAD

**Issue by MattRudy** [deldersveld/PowerBI-ThemeTemplates#64]

**Comment by MikeCarlo** [comment]
My recommendation would to not make issues for all of these, however instead pull a branch and make the appropriate changes and then ask for a pull request to submit your changes.  

Making many many comments without contributing what you know is not extremely helpful.

**Comment by MikeCarlo** [comment]
I'd also recommend using this tool to get the general properties as there is only really one general property. https://themes.powerbi.tips/properties

**Comment by MattRudy** [comment]
I agree Mike - I've been working to keep this repo updated and as I was going through templates I didn't realize just how many needed that added, I expected to create maybe 3-4 issues and not closer to 15.

I'm actively working on a fork to close these! But I figured I'd make the issues in case anyone else wanted to jump on some of them.

**Comment by MikeCarlo** [comment]
gotcha, thanks.

**Comment by MattRudy** [comment]
Also, love your site - I've used that tool before! I'll reach out to you separately with some questions about future features for the theme generator.
